mlir-sycl/include/mlir/Dialect/SYCL/IR/SYCLOps.td

```diff
@@ -481,9 +490,16 @@ def SYCLAccessorSubscriptOp
 
   let arguments = (ins Arg<AccessorMemRef, "The accessor", [MemRead]>:$Acc,
                        Arg<SYCLAccessorSubscriptIndex, "The offset", [MemRead]>:$Index);
-
   let results = (outs AnyType:$Res);
 
+  let extraClassDeclaration =  extraClassDeclarationBase # [{
```

Comment:
Here is how we append a couple of extra class declarations to the one in `extraClassDeclarationBase`.